<?xml version="1.0"?>
<launch>
    <rosparam file="$(find blob_tracking)/config/blob_tracking_config.yml" command="load"/>

    <node pkg="blob_tracking" type="blob_tracking_node.py" name="blob_tracking_node" ns="/wormbot" output="screen"/>
</launch>
